client/client.go

```diff
@@ -831,23 +832,28 @@ func (c *client) GetLocalTS(ctx context.Context, dcLocation string) (physical in
 }
 
 func (c *client) GetMinTS(ctx context.Context) (physical int64, logical int64, err error) {
-	tsoClient, serviceMode := c.getServiceClientProxy()
-	if tsoClient == nil {
-		return 0, 0, errs.ErrClientGetMinTSO.FastGenByArgs("tso client is nil")
+	protoClient := c.getClient()
+	if protoClient == nil {
+		return 0, 0, errs.ErrClientGetProtoClient
 	}
 
-	switch serviceMode {
```

Comment:
Why remove it?

Reply:
Since QPS is very low, decided to always let client query GetMinTS through the API leader. In this way, the logic is moved to the API leader's GetMinTS() server side implementation as shown below:
```
if s.IsAPIServiceMode() {
		minTS, err = s.GetMinTSFromTSOService(tso.GlobalDCLocation)
	} else {
		start := time.Now()
		ts, internalErr := s.tsoAllocatorManager.HandleRequest(tso.GlobalDCLocation, 1)
		if internalErr == nil {
			tsoHandleDuration.Observe(time.Since(start).Seconds())

   	         }
		minTS = &ts
	}
```

Reply:
I added some code back to handle compatibility issue in case of PD/API server doesn't support GetMinTS API